nms/app/theme/default.d.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @format
```

Comment:
Some of the files have the `@format`, some do not.
What is it good for? Can we have this consistent in all or non of the files?

Reply:
Copy paste error :( This is from prettier and has no effect since we don't use the require-pragma option https://prettier.io/docs/en/options.html#require-pragma

I'm trying to remove those together with the `@flow` during the migration.